bin/mainflux-launch.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+###
+# Launches all Mainflux Go binaries when they are installed globally.
```

Comment:
This does not make sense. If we are providing launcher, we should automate the installation, or at least provide flags to indicate whether we want something to be installed or not.

Reply:
Installation is automated via `Makefile`: https://github.com/mainflux/mainflux/pull/176

`make install` target will be added to install all the binaries.

Reply:
No it is not. If, as a user, I am enforced to install & configure everything, I don't see the point of providing an "automation" of easiest task of all - running 4 go binaries. Moreover, none of these will be configured properly (it will use the default configuration which might not be relevant).